node-bootstrapper/app.go

```diff
@@ -90,7 +103,7 @@ func (a *App) provisionStart(ctx context.Context, cse utils.SensitiveString) err
 	// it's also nice to have a single log file for all the important information, so write to both places
 	cmd.Stdout = io.MultiWriter(os.Stdout, &stdoutBuf)
 	cmd.Stderr = io.MultiWriter(os.Stderr, &stderrBuf)
-	err := a.cmdRunner(cmd)
+	_, err := a.cmdRunner(cmd)
```

Comment:
I don't think it's going to work.

`cmd.Output` overrides `Stdout` and `Stderr` properties.

I didn't use `Output` in this function mainly to allow streaming of shell script stdout/stderr for a potentially faster feedback. (Don't need to wait until cse execution finished before you can see the output)

Reply:
reverted it back